### ISSUE TITLE
Tensorflow slim validation accuracy is 0 for default models like mobilenet/inception trained on imagenet from scratch.

### ISSUE THREAD

**Issue by kakpple** [tensorflow/models#3372]
Hi,

As title says, 
I trained tensorflow slim default models like mobilenetv1 and inceptionv2 on imagenet from scratch.
The loss is decreasing from ~7 to ~2, training seems to be good.
But the validation accuracy using eval_image_classifier.py shows around 0.
Checkpoints trained from scratch is used for validation accuracy check.

While I use tensorflow provided pretrained checkpoints of mobilenetv1 to check validation accuracy, Accuracy is about 70% as claimed in the site.

I also trained darknet-19 from scratch and the validation accuracy is around 60%, and I added batch normalization to vgg16 and trained on imagenet, the validation accuracy is also above 50%.

Could anyone tell me why default models on slim like mobilenetv1 and inceptionv2 trained from scratch shows around 0 accuracy on validation?

Any ideas? 

Thanks in advance.

Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **What is the top-level directory of the model you are using**:
Tensorflow-slim
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Ubuntu 16.04
- **TensorFlow installed from (source or binary)**:
Binary
- **TensorFlow version (use command below)**:
1.4.1
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
CUDA 8.0 cuDNN 6.0
- **GPU model and memory**:
12G
- **Exact command to reproduce**:
Train mobilenetv1 on imagenet, and run eval_image_classifier.py for checkpoint trained from scratch

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by cy89** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
